common/log.hpp

```diff
@@ -0,0 +1,158 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#ifndef MENDER_LOG_HPP
+#define MENDER_LOG_HPP
+
+#include "config.h"
```

Comment:
I've voiced my opinion on includes, in this democracy I think it's up to you guys to decide now. 😉

Reply:
No strong opinion here. And using `#include <config.h>` is easier with no complications (for now), so it makes more sense to go that way.

Reply:
Noted, `<>` it is!